roles/check_resource/README.md

```diff
@@ -15,3 +15,33 @@ resource\_to\_check         | Yes       | MachineConfigPool      | Name of the r
 check\_wait\_retries        | Yes       | Undefined              | Number of times in which the wait task is performed.
 check\_wait\_delay          | Yes       | Undefined              | Time spent between wait tasks' iterations.
 check\_reason               | No        | Undefined              | Reason for the check to be done.
+
+## Requirements
+
+A running OpenShift cluster with the proper credentials is required, credentials must be passed as by setting the KUBECONFIG environment.
+
+## Example of usage
+
+Confirm that Machine Config Pools are not updating
+```
+- name: "Wait for updated MCP after applying ICSP"
+  include_role:
+    name: redhatci.ocp.check_resource
+  vars:
+    resource_to_check: "MachineConfigPool"
+    check_wait_retries: 120
+    check_wait_delay: 10
+    check_reason: "Apply ICSPs for mirrored catalogs"
+```
+
+Confirm that OVN pods are running
```

Comment:
Can we update the wording of this step? just to make it match what it does, it reads `OVN pods` but includes a check_resource for SriovNetworkNodeState, maybe something about "Confirming SRIOV node state.."

Reply:
Done